entities/communication/voiceCall.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
All properties are missing an example. You can add an example for each or add an example object at the end of the entity (the first is auto-generated if you use proper code annotations